lib/server.js

```diff
@@ -59,6 +60,9 @@ function serverFactory(core, messageHandler, Logger) {
             logger.info('proxyDHCP server is listening on %s:%s'.format(self.address, self.inPort));
             self.server.setBroadcast(true);
         });
+
+        var DHCPPoller = new IscDhcpLeasePoller({}, {});
+        DHCPPoller.run();
```

Comment:
That's the questions - what are we doing with the isc-dhcppoller. As the ARP Poller removed and disbled isc-dhcp-poller from on-tasksgraph. While this moves it from even being a job to being always on and part of on-dhcp-proxy.